beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -163,7 +163,7 @@ func (vs *Server) proposeGenericBeaconBlock(ctx context.Context, blk interfaces.
 	if sidecar != nil {
 		sidecarData = sidecar.Message
 		if err := vs.P2P.Broadcast(ctx, sidecar); err != nil {
-			log.Errorf("could not broadcast blobs sidecar: %v", err)
+			log.WithError(err).Error("could not broadcast blobs sidecar")
```

Comment:
Are these logging changes intentional / necessary?

Reply:
yeah. There's a linter in bazel that fails to build without these changes.